bip/t3dn_bip/formats.py

```diff
@@ -1,17 +1,59 @@
-import base64
-from collections import namedtuple
-
-BIP2_MAGIC = b'BIP2'
-
-_PillowFormat = namedtuple('_PillowFormat', ['magic', 'tests'])
-
-PILLOW_FORMATS = {
-    'png': _PillowFormat(
-        magic = b'\x89\x50\x4e\x47',
-        tests = [base64.b64decode(b'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABAQMAAAAl21bKAAAAA1BMVEUAAACnej3aAAAAAXRSTlMAQObYZgAAAApJREFUCNdjYAAAAAIAAeIhvDMAAAAASUVORK5CYII=')],
-    ),
-    'jpeg': _PillowFormat(
-        magic = b'\xff\xd8',
-        tests = [base64.b64decode(b'/9j/4AAQSkZJRgABAQEAYABgAAD/2wBDAAgGBgcGBQgHBwcJCQgKDBQNDAsLDBkSEw8UHRofHh0aHBwgJC4nICIsIxwcKDcpLDAxNDQ0Hyc5PTgyPC4zNDL/2wBDAQkJCQwLDBgNDRgyIRwhMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjL/wAARCAABAAEDASIAAhEBAxEB/8QAHwAAAQUBAQEBAQEAAAAAAAAAAAECAwQFBgcICQoL/8QAtRAAAgEDAwIEAwUFBAQAAAF9AQIDAAQRBRIhMUEGE1FhByJxFDKBkaEII0KxwRVS0fAkM2JyggkKFhcYGRolJicoKSo0NTY3ODk6Q0RFRkdISUpTVFVWV1hZWmNkZWZnaGlqc3R1dnd4eXqDhIWGh4iJipKTlJWWl5iZmqKjpKWmp6ipqrKztLW2t7i5usLDxMXGx8jJytLT1NXW19jZ2uHi4+Tl5ufo6erx8vP09fb3+Pn6/8QAHwEAAwEBAQEBAQEBAQAAAAAAAAECAwQFBgcICQoL/8QAtREAAgECBAQDBAcFBAQAAQJ3AAECAxEEBSExBhJBUQdhcRMiMoEIFEKRobHBCSMzUvAVYnLRChYkNOEl8RcYGRomJygpKjU2Nzg5OkNERUZHSElKU1RVVldYWVpjZGVmZ2hpanN0dXZ3eHl6goOEhYaHiImKkpOUlZaXmJmaoqOkpaanqKmqsrO0tba3uLm6wsPExcbHyMnK0tPU1dbX2Nna4uPk5ebn6Onq8vP09fb3+Pn6/9oADAMBAAIRAxEAPwD3+iiigD//2Q==')],
-    ),
+from io import BytesIO
+from base64 import b64decode
+
+
+class _Format:
```

Comment:
It feels weird, that we fine-tune the code. But we still use this class for the BIP formats even though it does not fit for the BIPs. If we want to save a class, we could just rename that class to `_PilFormat`; then rename `BIP_FORMATS` to `BIP_MAGICS` and change it to a dict, which just holds the magic as value.

Reply:
I split the class into _BIPFormat and _PILFormat, because I like keeping the structure the same.